src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx

```diff
@@ -95,24 +112,170 @@ To connect your frontend code to your backend, you need to:
 2. generate a new API client from the Amplify library
 3. make an API request with end-to-end type-safety
 
-In your app's entry point, typically **main.tsx** for React apps created using Vite, make the following edits:
+First, install the Amplify client library to your project:
+
+<InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
 
-```tsx
-// main.tsx
+```bash
+npm install aws-amplify
+```
+
+In your app's entry point, typically **main.tsx** for React apps created using Vite, make the following edits:
 
+```tsx title="src/main.tsx"
 import { Amplify } from 'aws-amplify';
 import config from '../amplifyconfiguration.json';
 
 Amplify.configure(config);
 ```
 
+</InlineFilter>
+
+<InlineFilter filters={["android"]}>
+
+Under Gradle Scripts, open build.gradle (Module :app), add the following lines:
+
+```groovy title="build.gradle.kts"
+dependencies {
+    // Amplify API dependencies
+    // highlight-start
+    implementation("com.amplifyframework:aws-api:2.14.11")
```

Comment:
These versions need to use the versions.ts so versions are updated all in one place. We also need to be careful in when we merge this due to not currently supporting amplifyoutputs.json until our gen2 release.